resources/views/layouts/table.blade.php

```diff
@@ -63,7 +63,10 @@
         </div>
     @endif
 
-    @includeWhen(($rows instanceof \Illuminate\Contracts\Pagination\Paginator || $rows instanceof \Illuminate\Contracts\Pagination\CursorPaginator) && $rows->isNotEmpty(),
+    @includeWhen(
```

Comment:
I think the condition is already very long. Let's try to remove it and indicate the missing part in `platform:: layouts.pagination`

Reply:
I think I don't get it. Why not just remove the condition? Isn't already indicated?